terraform/trial_rg/main.tf

```diff
@@ -30,6 +30,7 @@ resource "azurerm_app_service_plan" "apps_service_plan" {
 module "trial_audit" {
```

Comment:
I think the shared resource group also should include an env...

Reply:
its not failing the process - why add?

Reply:
I mean, when we create a new process.
we also want a new shared rg for that newly created env. we dont want to use the same shared rgs for all envs.